### ISSUE TITLE
2.12.0: pilot upload fails with the call to the iTMSTransporter failure

### ISSUE THREAD

**Issue by beefon** [fastlane/fastlane#14577]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description
`pilot upload` reports an error in iTMSTransporter:
`The call to the iTMSTransporter completed with a non-zero exit status: 1. This indicates a failure.`

##### Command executed
```
$ DELIVER_ITMSTRANSPORTER_ADDITIONAL_UPLOAD_PARAMETERS='-t DAV' \
FASTLANE_PASSWORD=XXXXXX FASTLANE_ITC_TEAM_ID=YYYY \
/usr/local/bin/fastlane _2.12.0_ pilot upload --ipa /Users/build/Desktop/App.ipa \
--beta_app_description 'Build for internal testing' \
--changelog 'Some changes' \
--distribute_external true \
--skip_waiting_for_build_processing true \
--skip_submission false --username xxx@xxx.com \
--groups '"Testers","Designers"' '
--verbose
```

##### Complete output when running fastlane, including the stack trace and command used

<details>

```
INFO [2019-04-12 16:03:04.34]: Login to iTunes Connect (xxx@xxx.com)
Loading session from '/Users/ios-build/.fastlane/spaceship/xxx@xxx.com/cookie'
INFO [2019-04-12 16:03:08.27]: Login successful
DEBUG [2019-04-12 16:03:08.33]: App identifier (x)
INFO [2019-04-12 16:03:08.93]: Ready to upload new build to TestFlight (App: x)...
DEBUG [2019-04-12 16:03:08.98]: App Platform (ios)
INFO [2019-04-12 16:03:09.82]: Wrote XML data to '/tmp/417281773.itmsp'
INFO [2019-04-12 16:03:09.98]: Going to upload updated app to iTunes Connect
INFO [2019-04-12 16:03:09.98]: This might take a few minutes. Please don't interrupt the script.
DEBUG [2019-04-12 16:03:10.11]: /Applications/Xcode_10_1.app/Contents/Applications/Application\ Loader.app/Contents/itms/java/bin/java -Djava.ext.dirs=/Applications/Xcode_10_1.app/Contents/Applications/Application\ Loader.app/Contents/itms/java/lib/ext -XX:NewSize=2m -Xms32m -Xmx1024m -Xms1024m -Djava.awt.headless=true -Dsun.net.http.retryPost=false -classpath /Applications/Xcode_10_1.app/Contents/Applications/Application\ Loader.app/Contents/itms/lib/itmstransporter-launcher.jar com.apple.transporter.Application -m upload -u xxx@xxx.com -p YourPassword -f /tmp/417281773.itmsp -t DAV -t Signiant -k 100000 2>&1
ERROR [2019-04-12 16:03:10.25]: The call to the iTMSTransporter completed with a non-zero exit status: 1. This indicates a failure.
/usr/local/lib/ruby/gems/2.4.0/gems/fastlane-2.12.0/fastlane_core/lib/fastlane_core/ui/interface.rb:145:in `user_error!': [!] Error uploading ipa file, for more information see above (FastlaneCore::Interface::FastlaneError)
	from /usr/local/lib/ruby/gems/2.4.0/gems/fastlane-2.12.0/fastlane_core/lib/fastlane_core/ui/ui.rb:14:in `method_missing'
	from /usr/local/lib/ruby/gems/2.4.0/gems/fastlane-2.12.0/pilot/lib/pilot/build_manager.rb:22:in `upload'
	from /usr/local/lib/ruby/gems/2.4.0/gems/fastlane-2.12.0/pilot/lib/pilot/commands_generator.rb:58:in `block (2 levels) in run'
	from /usr/local/lib/ruby/gems/2.4.0/gems/commander-4.4.6/lib/commander/command.rb:182:in `call'
	from /usr/local/lib/ruby/gems/2.4.0/gems/commander-4.4.6/lib/commander/command.rb:153:in `run'
	from /usr/local/lib/ruby/gems/2.4.0/gems/commander-4.4.6/lib/commander/runner.rb:446:in `run_active_command'
	from /usr/local/lib/ruby/gems/2.4.0/gems/fastlane-2.12.0/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:38:in `run!'
	from /usr/local/lib/ruby/gems/2.4.0/gems/commander-4.4.6/lib/commander/delegates.rb:15:in `run!'
	from /usr/local/lib/ruby/gems/2.4.0/gems/fastlane-2.12.0/pilot/lib/pilot/commands_generator.rb:134:in `run'
	from /usr/local/lib/ruby/gems/2.4.0/gems/fastlane-2.12.0/pilot/lib/pilot/commands_generator.rb:14:in `start'
	from /usr/local/lib/ruby/gems/2.4.0/gems/fastlane-2.12.0/fastlane/lib/fastlane/cli_tools_distributor.rb:45:in `take_off'
	from /usr/local/lib/ruby/gems/2.4.0/gems/fastlane-2.12.0/bin/fastlane:15:in `<top (required)>'
	from /usr/local/bin/fastlane:22:in `load'
	from /usr/local/bin/fastlane:22:in `<main>'
```
</details>

### Environment

<details><summary>🚫 fastlane environment 🚫</summary>

### Stack

| Key                         | Value                                            |
| --------------------------- | ------------------------------------------------ |
| OS                          | 10.13.6                                          |
| Ruby                        | 2.4.1                                            |
| Bundler?                    | false                                            |
| Git                         | git version 2.17.2 (Apple Git-113)               |
| Installation Source         | /usr/local/bin/fastlane                          |
| Host                        | Mac OS X 10.13.6 (17G4015)                       |
| Ruby Lib Dir                | /usr/local/Cellar/ruby/2.4.1_1/lib               |
| OpenSSL Version             | OpenSSL 1.0.2k  26 Jan 2017                      |
| Is contained                | false                                            |
| Is homebrew                 | false                                            |
| Is installed via Fabric.app | false                                            |
| Xcode Path                  | /Applications/Xcode_10_1.app/Contents/Developer/ |
| Xcode Version               | 10.1                                             |

### System Locale

| Error                       |
| --------------------------- |
| No Locale with UTF8 found 🚫 |

### fastlane files:

**No Fastfile found**

**No Appfile found**

### fastlane gems

| Gem      | Version | Update-Status      |
| -------- | ------- | ------------------ |
| fastlane | 2.108.0 | 🚫 Update available |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| did_you_mean              | 1.1.0        |
| slack-notifier            | 2.3.2        |
| atomos                    | 0.1.3        |
| CFPropertyList            | 2.3.5        |
| claide                    | 1.0.2        |
| colored2                  | 3.1.2        |
| nanaimo                   | 0.2.6        |
| xcodeproj                 | 1.7.0        |
| rouge                     | 2.0.7        |
| xcpretty                  | 0.3.0        |
| terminal-notifier         | 1.8.0        |
| unicode-display_width     | 1.3.0        |
| terminal-table            | 1.8.0        |
| plist                     | 3.3.0        |
| public_suffix             | 2.0.5        |
| addressable               | 2.5.1        |
| multipart-post            | 2.0.0        |
| word_wrap                 | 1.0.0        |
| tty-screen                | 0.6.4        |
| tty-cursor                | 0.5.0        |
| tty-spinner               | 0.8.0        |
| babosa                    | 1.0.2        |
| colored                   | 1.2          |
| highline                  | 1.7.8        |
| commander-fastlane        | 4.4.6        |
| excon                     | 0.57.1       |
| faraday                   | 0.12.1       |
| unf_ext                   | 0.0.7.4      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20170404 |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| fastimage                 | 2.1.0        |
| gh_inspector              | 1.1.2        |
| json                      | 2.0.2        |
| mini_magick               | 4.5.1        |
| multi_json                | 1.12.1       |
| multi_xml                 | 0.6.0        |
| rubyzip                   | 1.2.2        |
| security                  | 0.1.3        |
| xcpretty-travis-formatter | 0.0.4        |
| dotenv                    | 2.2.1        |
| bundler                   | 1.15.2       |
| faraday_middleware        | 0.11.0.1     |
| naturally                 | 2.1.0        |
| simctl                    | 1.6.4        |
| declarative               | 0.0.9        |
| declarative-option        | 0.1.0        |
| representable             | 3.0.4        |
| mime-types-data           | 3.2016.0521  |
| mime-types                | 3.1          |
| little-plugger            | 1.1.4        |
| logging                   | 2.2.2        |
| jwt                       | 1.5.6        |
| memoist                   | 0.16.0       |
| os                        | 0.9.6        |
| signet                    | 0.7.3        |
| googleauth                | 0.5.1        |
| httpclient                | 2.8.3        |
| google-api-client         | 0.21.2       |
| emoji_regex               | 0.1.1        |
| openssl                   | 2.0.3        |
| io-console                | 0.4.6        |
| retriable                 | 3.0.2        |
| psych                     | 2.2.2        |
| uber                      | 0.1.0        |
</details>

*generated on:* **2019-04-12**
</details>

**Comment by beefon** [comment]
I've been blocked by the 410-Gone-gate, now I've updated fastlane to the latest version (2.12.0?) and now `pilot upload` fails with the iTMS error.

**Comment by janpio** [comment]
Fastlane 2.12.0 is from January 2017, so not surprising stuff is broken :)

Please make sure you upgrade to fastlane 2.12_0_.0 and see if you still have this problem.

**Comment by beefon** [comment]
LOL. I've seen 2.12.0 somewhere recently, and used it. :(

**Comment by MoElnaggar14** [comment]
I have the same problem :(

**Comment by janpio** [comment]
You also installed fastlane 2.12.0 by accident, which can't work @MoElnaggar14?

**Comment by MoElnaggar14** [comment]
I updated last version yesterday before I uploading new version of my app so what should I do ? @janpio

**Comment by janpio** [comment]
Create a new bug issue and fill the template so we have all the necessary information please @MoElnaggar14.
